internal/service/device.go

```diff
@@ -75,19 +75,41 @@ func (h *ServiceHandler) ListDevices(ctx context.Context, request server.ListDev
 		return server.ListDevices400JSONResponse{Message: err.Error()}, nil
 	}
 
+	statusFilter := []string{}
+	if request.Params.StatusFilter != nil {
```

Comment:
We could already drop support for `statusFilter` here, given that's the old way of filtering for statuses and has been replaced by `fieldSelector`.

I'd expect we'll also remove the individual filter fields from the API soon, and keep only "fieldSelector" and "labelSelector".

Reply:
Hey @celdrake,
Removing this in this PR will break the functionality.
I agree with you about removing individual filter fields from the API, but this should be handled in a dedicated ticket that includes changes to the API, CLI, and backend functionality.

@amir-yogev-gh - FYI

Reply:
Sure, though I will already include the change in the UI PR (which hasn't been merged yet), to use "fieldSelector" instead of statusFilter.